css/index.css

```diff
@@ -135,12 +173,93 @@ footer nav {
     width: 60%;
     display: flex;
     justify-content: space-between;
-    align-items: center;
+    align-items: left;
     padding: 20px 2%;
     font-size: 14px;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
+	margin-right:20px
+}
+
+@media (max-width: 1000px) {
+	.middle-content .boxes .box.box2{
+		background-color:forestgreen;
+	}
+	.middle-content .boxes .box.box4{
+		background-color:darkorchid;
+	}
+	.middle-content .boxes .box.box6{
+		background-color:hotpink;
+	}
+	.middle-content .boxes .box.box8{
+		background-color:indigo;
+	}
+	.middle-content .boxes .box.box10{
+		background-color:limegreen;
+	}
+}
+
+@media (max-width: 768px) {
+	header{
+		flex-wrap:wrap;
+		justify-content:center;
+	}
+	.logo{
+		margin-bottom:4px;
+	}
+	nav{
+		margin-bottom:33px;
+	}
+	.middle-content .boxes .box.box1{
+		background-color:teal;
+	}
+	.middle-content .boxes .box.box3{
+		background-color:gold;
+	}
+	.middle-content .boxes .box.box5{
+		background-color:cadetblue;
+	}
+	.middle-content .boxes .box.box7{
+		background-color:coral;
+	}
+		.middle-content .boxes .box.box9{
+		background-color:crimson;
+	}
+}
+
+@media (max-width: 400px) {
+	.middle-content .boxes .box.box10{
+	display:none;
+	}
+	.middle-content .boxes{
+		display:flex;
+		flex-direction:row-reverse;
```

Comment:
No need to repeat `display: flex` here since it's already declared above. You're just overriding `flex-direction`. Nice work otherwise! ☺️